src/bundle/Resources/public/scss/_mixins.scss

```diff
@@ -252,6 +252,10 @@
         width: 100%;
         padding: calculateRem(8px) calculateRem(24px) calculateRem(24px) calculateRem(24px);
 
+        &:focus {
```

Comment:
Please, provide a different styling approach for focused elements. For better accessibility.